packages/server/__tests__/db/seeds/fixtures.js

```diff
@@ -35,7 +35,7 @@ const agencies = {
         abbreviation: 'Sub SBA',
         code: 'SSBA',
         name: 'State Board of Sub Accountancy',
-        parent: 0,
+        parent: tenants.SBA.id,
```

Comment:
Doesn't look like it would cause issues for now, but given that `agencies.parent` is a reference to `agencies.id` (not `tenants.id`), this might result in a hard-to-spot error down the road. 

I'd suggest either reverting this change or breaking it out of the `const agencies` definition so that it can be set to `agencies.accountancy.id`.

Reply:
good catch - I've updated/replaced the old ID and fixed another test issue.